### PR TITLE
Added enabled flag for actuator

### PR DESCRIPTION
**Pull request by triccyx**
It is possibile to deactivate the actuator by the config file.

### CODE REVIEW

File: plugins/actuator/FMIActuatorPlugin.cc [hunk 1]
```diff
@@ -85,6 +85,13 @@ void FMIActuatorPlugin::Load(gazebo::physics::ModelPtr _parent, sdf::ElementPtr
               << std::endl;
     }
 
+    if(!actuator.m_enabled)
+    {
+        gzwarn << "FMIActuatorPlugin: actuator disabled:"<<actuator.name
```
Comment (traversaro) [inline plugins/actuator/FMIActuatorPlugin.cc]:
```suggestion
        gzwarn << "FMIActuatorPlugin: actuator disabled: "<<actuator.name
```

File: plugins/actuator/README.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ The plugin filename is `libFMIActuatorPlugin.so` .
 | fmu            | string  | Filename of the FMU plugin to use for actuator co-simulation. | Yes | This name is passed to the [`gazebo::common::SystemPaths::FindFile`](http://osrf-distributions.s3.amazonaws.com/gazebo/api/9.0.0/classgazebo_1_1common_1_1SystemPaths.html#a9e03f07eac9f89d8c4c14af5660fa938) method to find the absolute location of the FMU file. Adding the directory containing the FMUs to the [`GAZEBO_RESOURCE_PATH`](http://gazebosim.org/tutorials?tut=components) should be sufficient to make it visible to the plugin. |
 | disable_velocity_effort_limits   | bool | True if the joint velocity and effort limits are disabled (default: false). | No |  This is useful if the transmission input is in a unit completly different from N or Nm, and the effort limits will be completly unrealisting for the actuator input. |
 | variable_names | composite element | Optional tag to specify if the input/output variable names in FMU are different from the default ones. | No | |
+| enabled | bool | Enable or disable the actuator. | No | Default value true |
```
Comment (traversaro) [inline plugins/actuator/README.md]:
```suggestion
| enabled | bool | Enable or disable the actuator. When an actuator is not enabled, the simulation ignores it, and it behaves like as if the actuator was not present at all in the SDF. | No | Default value: true |
```

**Comment by traversaro** [comment]
For making sure that there is no problem missed by the reviewer, it is typically better to wait for the Travis output before merging a branch. @triccyx

**Comment by triccyx** [comment]
Oops sorry!

**Comment by traversaro** [comment]
No problem, I just added a check in the GitHub configuration so it is harder to merge a PR if Travis did not pass!

**Pull request merged**